### DISCUSSION TITLE
Is there a way to use "column-reverse" for reverse-infinite scroll, instead of the whole prepending elements thing?

### DISCUSSION THREAD

**Discussion by decipher-cs** [petyosi/react-virtuoso#1020] [Q&A]
I was still confused about how to make the `firstItemIndex` work for reverse infinite scrolling even after looking at <https://virtuoso.dev/prepend-items/>.

 So I thought I'd ask if something like flexbox's `flex-direction: 'column-reverse` would work for reversing the order of the elements. This would really decrease the complexity cause then I can just normally append items and CSS will handle the whole reversing part. 
```jsx
<Virtuoso style={{ display: 'flex', flexDirection: 'column-reverse' }}/>
```
Not sure if that would mess with the virtualization algorithm or not so I thought I'd just ask.

**Comment by Zerebokep** [comment]
I have the same q.
